rules/false_positives/nmap.yara

```diff
@@ -13,3 +13,16 @@ rule nmap_fingerprints: override {
   condition:
     filesize < 512KB and $description and $license and #fingerprint > 0
 }
+
+rule nping_bin: override {
```

Comment:
This override seems to match the rule it's overriding. I feel like "low" may not be the right outcome, given that someone wrote the rule for it. Maybe MEDIUM?

Reply:
Good call.

Reply:
Updated in [`94f75d4` (#838)](https://github.com/chainguard-dev/malcontent/pull/838/commits/94f75d4a846047ff3d98be23b379eb9d577c6407).